__tests__/useLazyLoadQueryNode-test.tsx

```diff
@@ -155,7 +159,9 @@ describe('useLazyLoadQueryNode', () => {
         gqlNoNameQuery = generated.UserNoNameQuery;
         variables = { id: '1' };
         query = createOperationDescriptor(gqlQuery, variables);
-        renderFn = jest.fn((result) => result?.node?.name ?? 'Empty');
+        renderFn = jest.fn((result) =>
+            result && result.node && result.node.name ? result.node.name : 'Empty',
+        );
```

Comment:
Optional chaining can’t be used?

Reply:
we could by updating typescript. But I preferred to avoid doing it in this PR